valhalla/baldr/graphreader.h

```diff
@@ -793,6 +794,52 @@ class GraphReader {
    */
   int GetTimezone(const baldr::GraphId& node, const GraphTile*& tile);
 
+  // TODO: temporary until we have real incidents loaded from incident loading singleton
+  struct Incident {
+    uint32_t edge_index;
+    double start_offset;
+    double end_offset;
+    uint64_t id;
+  };
+  using incident_tile_t = std::vector<Incident>;
+
+  /**
+   * Returns an incident tile for the given tile id
+   * @param tile_id  the tile id for which incidents should be returned
+   * @return the incident tile for the tile id
+   */
+  virtual std::shared_ptr<incident_tile_t> GetIncidentTile(const GraphId& tile_id) const {
```

Comment:
this method gets overriden in a test graph reader in my unit test so taht i can inject incidents directly into the reader on the exact edges exactly where i want them to be. made the testing very very nice.